Mathlib/MeasureTheory/Function/ConditionalExpectation/Basic.lean

```diff
@@ -387,34 +395,46 @@ lemma Memℒp.condExpL2_ae_eq_condExp (hm : m ≤ m₀) (hf : Memℒp f 2 μ) [I
     condExpL2 E 𝕜 hm hf.toLp =ᵐ[μ] μ[f | m] :=
   hf.condExpL2_ae_eq_condExp' hm (memℒp_one_iff_integrable.1 <| hf.mono_exponent one_le_two)
 
+end RCLike
+
+section Real
+variable [InnerProductSpace ℝ E]
+
 -- TODO: Generalize via the conditional Jensen inequality
-include 𝕜 in
 lemma eLpNorm_condExp_le : eLpNorm (μ[f | m]) 2 μ ≤ eLpNorm f 2 μ := by
```

Comment:
Are you certain that you want to un-generalise this lemma? It seems more useful to make `𝕜` explicit here, rather than specialising the statement

Reply:
Any `𝕜`-inner product space for `RCLike 𝕜` is a real inner product space (see [`InnerProductSpace.rclikeToReal`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/InnerProductSpace/Basic.html#InnerProductSpace.rclikeToReal), so this is no loss in generality

Furthermore, this "ungeneralisation" is not a new pattern. Searching for `InnerProductSpace ℝ` yields 112 hits in 57 files